### ISSUE TITLE
Bug Report: thread panicked while processing panic. aborting.

### ISSUE THREAD

**Issue by zackshen** [aptabase/tauri-plugin-aptabase#9]
I simulated a panic in the tauri command. but the `with_panic_hook` callback handler caused panic again. I setup a dead simple `with_panic_hook` which just print some plain text.  i found `lib.rs` #74 `hook_client.flush_blocking();` may create panic again.

```rust
#[tauri::command]
pub async fn update(
    app_handle: tauri::AppHandle,
) -> Result<(), Errors> {
    panic!("make a panic to test the error handler");
}
```

## Environment:
* tauri-plugin-aptabase: 0.4
* tauri: 1.5.x
* os: MacOS 13.5

**Comment by zackshen** [comment]
i found `tauri-plugin-aptabase` released 0.5.0, the bug has been fixed.
but no `0.5.0` version in the release page and changelog.md. 😭

**Comment by goenning** [comment]
good catch @zackshen :)

I completely forgot about those steps. Thank you, I fixed it :)
